### PR TITLE
Fix/actuator tests

### PR DESCRIPTION
**Pull request by simenbootsma**
Most of the PuTTY commands work again

### CODE REVIEW

File: Core/Src/ballSensor.c [hunk 1]
```diff
@@ -75,7 +75,7 @@ uint8_t notification_touch[] = {BS_NOTIFICATION, 17, BS_DEVICE_ADDRESS_AIR, BS_T
 // ====================== PUBLIC FUNCTIONS ====================== //
 
 bool ballSensor_Init(){
-	Putty_printf("[ballsensor] init\n");
+//	Putty_printf("[ballsensor] init\n");
```
Comment (emielsteerneman) [inline Core/Src/ballSensor.c:78]: please uncomment this

**Review by emielsteerneman — COMMENTED** [review]
I've tested it and for me, on Linux if that's relevant, only my first command is being executed. E.g. if I first run "dribbler 50", it will start spinning, but if I then run anything else, the robot won't react to it.

On another note, I'm not a big fan of having all these if-statements spread throughout the code. The robot can now be controlled from different locations in the code, instead of just the main loop. This makes it harder to both understand and modify the code flow. I'd rather see the new message protocol be extended to support testing, in such a way that everything goes via the loop() function.

File: Core/Src/ballSensor.c (hunk 1 above)
Comment (simenbootsma) [reply Core/Src/ballSensor.c:78]: Strange, it worked fine for me on Windows and for someone else on Linux. Could you maybe try once more?

I agree that it would be a lot better to have testing go via the loop() function as well. Perhaps this could be dealt with in a different PR.

The ballsensor is initialized every second when it is not connected or not working properly. That print is therefore quite annoying when you're trying to type something in Putty. Isn't the print in robot.c > init() sufficient?

```
      if(ballSensor_Init())
		Putty_printf("Ballsensor initialized\n");	
```

Comment (emielsteerneman) [reply Core/Src/ballSensor.c:78]: Ah of course, I forgot that most robots don't have a ballsensor connected, resulting in spam. I think we can test most of the functionality using the testRobot.py script. I'm planning on extending REM to include sending text messages as well. 

It's indeed weird that it does not work on Linux. It might have something to do with line endings. Windows sends \r\n, whereas Linux just sends \n if I'm correct. I might have some time to test this on friday.

**Review by emielsteerneman — APPROVED** [review]
LGTM; 

Again, I really dislike weird edge cases and side roads in our code flow, such as 
```
if( halt && !test_isTestRunning(square) )
```
but for the sake of development, let's merge it
